### PR TITLE
OPENEUROPA-2943: Upgrade to PHP 7.3.

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

[Insert description here]

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Comment by nagyad** [comment]
Follow up ticket for the problem you found?

File: docker-compose.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 version: '2'
 services:
   web:
-    image: fpfis/httpd-php-dev:7.1
+    image: fpfis/httpd-php-ci:7.3
```
Comment (ademarco) [inline docker-compose.yml]: This should be
```
     image: fpfis/httpd-php-dev:7.3
```

File: modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/oe_webtools_analytics_rules.install [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<?php
+
+/**
+ * @file
+ * Contains install and update functions for OpenEuropa Webtools Analytics Rules.
```
Comment (nagyad) [inline modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/oe_webtools_analytics_rules.install:5]: Over 80 chars.

File: modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/oe_webtools_analytics_rules.install [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+ */
+
+declare(strict_types = 1);
+
+use Drupal\Core\Cache\Cache;
```
Comment (nagyad) [inline modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/oe_webtools_analytics_rules.install:10]: Unused statement.

File: modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/oe_webtools_analytics_rules.install [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+
+/**
+ * Implements hook_install().
+ */
+function oe_webtools_analytics_rules_install() {
+  // Because the OpenEuropa Webtools Analytics Rules module contains
+  // event subscriber which use cache context 'url.path', we must invalidate
+  // all cached HTML output.
+  Cache::invalidateTags(['rendered']);
+  \Drupal::service('twig')->invalidate();
```
Comment (ademarco) [inline modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/oe_webtools_analytics_rules.install:20]: I'm not sure about this, we don't clear caches in `hook_install()`s. Is this module enabled optionally by Behat by any chance, when we run the test?

Comment (sergepavle) [reply modules/oe_webtools_analytics/modules/oe_webtools_analytics_rules/oe_webtools_analytics_rules.install:20]: Yes. for oe_webtools we are doing enabling modules per Behat feature.
About clearing cache on install, in core some modules also doing a clearing of render cache on install like here: https://github.com/drupal/core/blob/8.8.x/modules/block/block.install#L17 and here: https://github.com/drupal/core/blob/8.8.x/modules/layout_builder/layout_builder.install#L40.

**Pull request merged**